### DISCUSSION TITLE
first line / last line navigation stopped working inside vscode after recent lazyvim update

### DISCUSSION THREAD

**Discussion by ECon65** [LazyVim/LazyVim#5149] [General]
I'm not sure whether I should open an issue or I've missed something but after recent lazyvim update to v14. gg and G navigation stopped working when using neovim with vscode.
Using gg or G makes a jump but only few lines around.

**Comment by dpetka2001** [comment]
There were some changes made yesterday in the vscode Extra to disable animations and some other snacks stuff because they were causing weird behavior, but those changes will be available on the next stable release. Or if you want to follow HEAD so that you can get access to those changes immediately, you can follow the [docs](https://www.lazyvim.org/configuration/lazy.nvim).

**Reply by ECon65** [reply]
Thank you for your quick answer. Currently it is not critical for me so I will wait few more days. If still not resolved I will switch to HEAD.

**Comment by ECon65** [comment]
issue is resolved by upgrading to 14.5
